### PR TITLE
Prepare Nextcloud 30 release

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: .github/workflows/phpunit-sqlite.yml [hunk 1]
```diff
@@ -42,20 +61,15 @@ jobs:
...
-        server-versions: ['master']
-        include:
-          - php-versions: '8.3'
-            server-versions: 'stable29'
-          - php-versions: '8.3'
-            server-versions: 'stable28'
-          - php-versions: '8.0'
-            server-versions: 'stable27'
+        php-versions: ${{ fromJson(needs.matrix.outputs.php-version) }}
+        server-versions: ${{ fromJson(needs.matrix.outputs.server-max) }}
```
Comment (mejo-) [inline .github/workflows/phpunit-sqlite.yml:70]: This means phpunit will only run on latest server release and no longer on `stable28` and `stable29`, right? I'm fine with this change, just wanted to make sure it's on purpose.

**Review by mejo- — APPROVED** [review]
Thanks :heart: 

Changes look good to me - only one comment.

**Comment by mejo-** [comment]
Seems like we'll have to adjust the merge requirements because the expected `Nextcloud` CI job doesn't exist anymore. It's named `static-psalm-analysis` now.

File: .github/workflows/phpunit-sqlite.yml (hunk 1 above)
Comment (nickvergessen) [reply .github/workflows/phpunit-sqlite.yml:70]: that's right. mysql would test the full matrix it seems. Let me add it

**Comment by nickvergessen** [comment]
> Seems like we'll have to adjust the merge requirements because the expected Nextcloud CI job doesn't exist anymore. It's named static-psalm-analysis now.

Already fixed

**Comment by nickvergessen** [comment]
Also required mysql now, that runs php 8.3 against all servers and all php versions against master

**Pull request merged**